.github/workflows/ci-pipeline.yml

```diff
@@ -402,7 +432,7 @@ jobs:
         ./freec                                          \
           --transform-pattern-matching                   \
           --dump-transformed-modules example/transformed \
-          -b ./base                                      \
+          -b ./base/coq                                  \
```

Comment:
It's a bit weird that the user has to point to the Coq base library even if they want to compile to Agda. The `*.toml` files should be moved to `./base` or there should be another set of `*.toml` files in `./base/agda`.